apps/native/app/src/ui/lib/card/license-list-card.tsx

```diff
@@ -0,0 +1,275 @@
+  width: 42px;
+  height: 42px;
+  border-radius: ${({ theme: { border } }) => border.radius.large};
+`
+
+const Title = styled(Typography)<{ color: string }>`
+  margin-bottom: ${({ theme }) => theme.spacing.smallGutter}px;
+`
+
+const ImgWrap = styled.View`
+  flex-shrink: 0;
+  align-content: center;
+  justify-content: center;
+  height: 42px;
+  width: 42px;
+`
+
+export const LicenseCardPresets: Record<LicenseType, CardPreset> = {
+  DriversLicense: {
+    title: 'Ökuskírteini (IS)',
+    logo: LogoCoatOfArms,
+    backgroundImage: BackgroundDriversLicense,
+    backgroundColor: '#F5E4EC',
+    barcode: {
+      background: '#F5EAEF',
+      overlay: '#F0DDE5',
+    },
+  },
+  AdrLicense: {
+    title: 'ADR skírteini',
+    logo: LogoAOSH,
+    backgroundImage: BackgroundADR,
+    backgroundColor: '#F2FAEC',
+    barcode: {
+      background: '#FAFDF7',
+      overlay: '#F4FCEE',
+    },
+  },
+  MachineLicense: {
+    title: 'Vinnuvélaskírteini',
+    logo: LogoAOSH,
+    backgroundImage: BackgroundVinnuvelar,
+    backgroundColor: '#C5E6AF',
+    barcode: {
+      background: '#DEF1D1',
+      overlay: '#C8E6B3',
+    },
+  },
+  FirearmLicense: {
+    title: 'Skotvopnaleyfi',
+    logo: CoatOfArms,
+    backgroundImage: BackgroundWeaponLicense,
+    backgroundColor: '#EBEBF2',
+    barcode: {
+      background: '#FDFDF7',
+      overlay: '#FAFAEB',
+    },
+  },
+  HuntingLicense: {
+    title: 'Veiðikort',
+    logo: LogoEnvironmentAgency,
+    backgroundImage: BackgroundHuntingCard,
+    backgroundColor: '#E2EDFF',
+    barcode: {
+      background: '#DBECF4',
+      overlay: '#B3D3E3',
+    },
+  },
+  Ehic: {
+    title: 'Evrópska sjúkratryggingakortið',
+    logo: LogoEhic,
+    backgroundImage: BackgroundPassport,
+    backgroundColor: '#E2EDFF',
+  },
+  DisabilityLicense: {
+    title: 'Örorkuskírteini',
+    logo: DisabilityLicenseLogo,
+    backgroundImage: DisabilityLicenseBg,
+    backgroundColor: '#C5D5C8',
+    barcode: {
+      background: '#D7E3D7',
+      overlay: '#A0BAA2',
+    },
+  },
+  PCard: {
+    title: 'Stæðiskort',
+    logo: LogoCoatOfArms,
+    backgroundImage: BackgroundPCardLicense,
+    backgroundColor: '#F2F7FF',
+  },
+  Passport: {
+    title: 'Almennt vegabréf',
+    logo: LogoRegistersIceland,
+    backgroundImage: BackgroundPassport,
+    backgroundColor: '#fff',
+  },
+}
+
+export enum CustomLicenseType {
+  Passport = 'Passport',
+}
+
+type LicenseType = GenericLicenseType | CustomLicenseType
+
+type CardPreset = {
+  title: string
+  logo: ImageSourcePropType
+  backgroundImage: ImageSourcePropType
+  backgroundColor: string
+  barcode?: {
+    background: string
+    overlay: string
+  }
+}
+
+interface LicenseCardProps {
+  title?: string
+  nativeID?: string
+  style?: StyleProp<ViewStyle>
+  type?: LicenseType
+  logo?: ImageSourcePropType | string
+  backgroundImage?: ImageSourcePropType
+  backgroundColor?: string
+  licenseNumber?: string
+}
+
+export function LicenseListCard({
```

Comment:
Maybe a LicenseItemCard is more appropriate name 🤷‍♂️,  since list indicates that this component is a list not a list item